src/main/java/io/ebean/annotation/DbTablespace.java

```diff
@@ -0,0 +1,34 @@
+package io.ebean.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Specify in which tablespaces the data (tables, indices and lobs) will be stored. The annotation can be used for entities.
+ * <p>
+ * This is currently only useful for DB2 and not used for any other database.
+ * </p>
+ */
+@Target({ElementType.TYPE, ElementType.ANNOTATION_TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface DbTablespace {
```

Comment:
Slightly odd but I think we should call this `Tablespace` rather than `DbTablespace` (along the lines that we have StorageEngine). I realise we have a bunch of `Db*` annotations but yeah I think this should just be `Tablespace` in this case.